### ISSUE TITLE
How to use timevis with HTML ui?

### ISSUE THREAD

**Issue by emillykkejensen** [daattali/timevis#23]
What should I include in <head> to use timevis with HTML UI and what classes should I use?

**Comment by daattali** [comment]
I haven't tried - so I wouldn't know without spending some time trying it out. I assume it would work the same as including any other htmlwidget. If you've ever included a different htmlwidget and it worked fine, and timevis isn't working, then let me know. If you haven't done it with any, then perhaps this question should be asked on stackoverflow or somewhere else that's not specific to timevis

**Comment by emillykkejensen** [comment]
Thanks for the fast reply. I have included other htmlwidget, but for some reason this didn’t work like expected – however I did make it work in the end.
What I need to include is these two css libraries:
```
<link href="vis-4.16.1/vis.min.css" rel="stylesheet" />
<link href="timeline-0.4/timevis.css" rel="stylesheet" />
```
And js libraries:
```
<script src="htmlwidgets-0.8/htmlwidgets.js"></script>
<script src="vis-4.16.1/vis.min.js"></script>
<script src="timevis-binding-0.4/timevis.js"></script>
```
Normally I would include both the css and js library in the head section of the html, but for some reason timevis only works, if I include the js part in the body section.
Strange - maybe it’s my html theme that messes something up, but thought it would be nice to know for others, as it took me some time to get to.
